src/Tizen.Telephony/Tizen.Telephony/Call.cs

```diff
@@ -26,6 +26,7 @@ namespace Tizen.Telephony
     /// It provides the list of CallHandle which can be used to get the information about call related actions.
     /// </summary>
     /// <since_tizen> 3 </since_tizen>
+    /// [Obsolete("Deprecated since API13, will be removed in API15.")]
```

Comment:
[Obsolete Attribute](https://learn.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-9.0) is a C# language construct rather than a part of documentation/description. Please remove triple slash from those lines.
```suggestion
    [Obsolete("Deprecated since API13, will be removed in API15.")]
```

Reply:
Modified.